tls-certs/ca.crt

```diff
@@ -0,0 +1,29 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Can you please remove the certificates from this PR completley? When i merge this after the review comments above i will generate a set of certificates myself from your docs you provided and commit them to the repo so they are verified to come from me personally. Also when you do this then either remove the file additions completley from the x3 PR commits you have in your branch so it won't be first an addition and then one removal. If not then just commit the removal as a seperate commit and i will squash rebase merge this when i merge the PR.
Do this for all 4 certificate files please.